.buildkite/pipeline.yml

```diff
@@ -0,0 +1,22 @@
+steps:
+  - label: "Julia v1.10"
+    plugins:
+      - JuliaCI/julia#v1:
+          version: "1.10"
+      - JuliaCI/julia-test#v1: ~
+    agents:
+      queue: "juliagpu"
+      cuda: "*"
```

Comment:
we don't need nvidia GPUs for MadNLPTesting (only needed for MadNLPGPU). Wouldn't this be waste of resources?

Reply:
It's a waste of ressources but I don't in which subpackages you are using `GPU`.

Reply:
We only need it for testing `MadNLPGPU`